kpatch-build/create-diff-object.c

```diff
@@ -239,7 +239,7 @@ static void kpatch_bundle_symbols(struct kpatch_elf *kelf)
 
 	list_for_each_entry(sym, &kelf->symbols, list) {
 		if (is_bundleable(sym)) {
-			if (sym->sym.st_value != 0 &&
+			if (sym->sym.st_value != 0 && !sym->prefix_link &&
 			    !is_gcc6_localentry_bundled_sym(kelf, sym)) {
 				ERROR("symbol %s at offset %lu within section %s, expected 0",
 				      sym->name, sym->sym.st_value,
```

Comment:
This warning probably needs to be updated, i.e. shouldn't we warn about a bundled prefixed symbol which is not at 16 bytes, or a bundled localentry symbol which is not at 8 bytes?

Reply:
Yea, there's a bunch of different conditions that are now crammed in here that should be called out.